jest.base-config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   rootDir: __dirname,
   setupFilesAfterEnv: ['<rootDir>/test/unit.setup.js'],
   modulePathIgnorePatterns: ['.cache', 'dist'],
-  testPathIgnorePatterns: ['.testdata.js'],
+  testPathIgnorePatterns: ['.testdata.js', '.test.utils.js'],
```

Comment:
This seemed to be the smallest change to still work with the testMatch line below, although in the future we should modify that to only match `.test.[jt]s?(x)` files instead of all files, and then ignore any other files in __test__.

Reply:
Make sense, could you add a TODO comment that reflects what you said here in the code so that we remember to update it?